src/screens/transactions/chart/TransactionChart.tsx

```diff
@@ -115,8 +115,11 @@ export default class TransactionChart extends BaseAutoRefreshScreen<Props, State
 
   public getChargingStation = async (chargingStationID: string): Promise<ChargingStation> => {
     try {
+      const extraParams = {
```

Comment:
Should be typed HttpChargingStationRequest

`const extraParams: HttpChargingStationRequest = {...`